packages/xo-server/src/api/host.js

```diff
@@ -5,13 +5,21 @@ import { format, JsonRpcError } from 'json-rpc-peer'
 export async function set({
   host,
 
+  iscsiIqn,
   multipathing,
   name_label: nameLabel,
   name_description: nameDescription,
 }) {
   host = this.getXapiObject(host)
 
   await Promise.all([
+    iscsiIqn === undefined
+      ? false
```

Comment:
I think it's better to use the pattern `cond && expr` here because we don't want to return a value depending of the condition, we want to do an action conditionally, ideally we would use an `if` but we can't because this needs to be an expression.